.github/workflows/ml-kem.yml

```diff
@@ -69,6 +70,23 @@ jobs:
       - run: cargo test
       - run: cargo test --all-features
 
+  bench:
+    needs: set-msrv
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: RustCrypto/actions/cargo-cache@master
+      - uses: dtolnay/rust-toolchain@master
+        with:
+          toolchain: stable
+      - run: cargo install cargo-codspeed
```

Comment:
This will compile `cargo-codspeed` on each CI run, which is really inefficient. It's better to use pre-compiled releases, e.g. see how we [handle](https://github.com/RustCrypto/actions/blob/master/cargo-hack-install/action.yml) `cargo-hack` installation.

Reply:
Thanks, will try to get things working and then optimize :)